e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/filecreationissue/FileCreationIssueEMTest.java

```diff
@@ -0,0 +1,174 @@
+
+            args.add("--blackBox");
+            args.add("true");
+            args.add("--bbTargetUrl");
+            args.add(baseUrlOfSut);
+            args.add("--bbSwaggerUrl");
+            args.add(baseUrlOfSut + "/v2/api-docs");
+            args.add("--outputFormat");
+            args.add("JAVA_JUNIT_4");
+            args.add("--outputFolder");
+            args.add(outFolderName);
+            args.add("--executiveSummary");
+            args.add("true");
+            args.add("--maxTime");
+            args.add("30s");
+
+            // run evomaster
+            initAndRun(args);
+
+            // check that the folder org/foo/FileCreationIssue contains the file: EvoMaster_fault_representatives_Test
+            // and does not contain the file EvoMaster_Test.java
+            String currentDir = System.getProperty("user.dir");
+
+            String filePath = currentDir + "/" + outFolderName;
+
+            outFolder = new File(filePath);
+
+            Assert.assertTrue(outFolder.exists() && outFolder.isDirectory());
+
+            String files[] = outFolder.list();
+
+            boolean summaryFound = false;
+            boolean emptyFound = false;
+
+            for (String s : files) {
+                if (s.equals("EvoMaster_fault_representatives_Test.java")) {
+                    summaryFound = true;
+                } else if (s.equals("EvoMaster_Test.java")) {
+                    emptyFound = true;
+                }
+            }
+
+            // summary should be found, empty should not be found
+            Assert.assertTrue(summaryFound);
+            Assert.assertFalse(emptyFound);
+        } catch(Exception e) {
+            e.printStackTrace();
+        } finally {
+            if (outFolder != null) {
+                if(outFolder.exists()) {
+                    FileUtils.deleteDirectory(outFolder);
+                }
+            }
+        }
+
+
+    }
+
+    /**
+     * If executive summary is not requested, there should not be a file with name
+     * EvoMaster_fault_representatives_Test.java, and there should not be a file
+     * with name EvoMaster_Test.java
+     * @throws Throwable
+     */
+    @Test
+    public void testFileCreationIssueWithoutSummary() throws Throwable {
+
+        File outFolder = null;
+
+        try {
+
+            String outFolderName = "fileCreationIssueTest";
+
+            List args = new ArrayList<String>();
+
+            args.add("--blackBox");
+            args.add("true");
+            args.add("--bbTargetUrl");
+            args.add(baseUrlOfSut);
+            args.add("--bbSwaggerUrl");
+            args.add(baseUrlOfSut + "/v2/api-docs");
+            args.add("--outputFormat");
+            args.add("JAVA_JUNIT_4");
+            args.add("--outputFolder");
+            args.add(outFolderName);
+            args.add("--executiveSummary");
+            args.add("false");
+            args.add("--maxTime");
+            args.add("30s");
```

Comment:
same comment as previous test

Reply:
Hi Andrea,

Thanks for your comment. I have made it 5 seconds since our only goal is to see files created in this test case. 5 seconds should be enough for this in my opinion.